wagtail/core/tests/test_page_model.py

```diff
@@ -1484,6 +1484,12 @@ def test_copy_page_with_excluded_parental_and_child_relations(self):
             # reset excluded fields for future tests
             EventPage.exclude_fields_in_copy = []
 
+    def test_copy_unsaved_page(self):
+        """Test that unsaved page will not be copied."""
+        new_page = SimplePage.objects.get(url_path='/home/about-us/')
+        with self.assertRaises(Exception):
```

Comment:
Just need to update this line to check for the `RuntimeError` then this is good to go!